meta/boardreports/2022-11.md

```diff
@@ -0,0 +1,40 @@
+# Patterns Board Report (Nov)
+
+## Meta
+
+* Reporting Period: August 1st, 2022 - October 31st, 2022
+* [merged PRs](https://github.com/InnerSourceCommons/InnerSourcePatterns/pulls?q=is%3Apr+closed%3A2022-08-01..2022-10-31+is%3Amerged)
+* [opened issues](https://github.com/InnerSourceCommons/InnerSourcePatterns/issues?q=is%3Aissue+created%3A2022-08-01..2022-10-31+is%3Aopen+)
+
+## Engagement
+
+The [patterns book] is the way inner source practices are captured and shared. Recent web analytics:
+
+* minor traffic increase (4%) on the patterns book
+   * 12,627 page views total (previous report was 12,099)
+   * ~140 per day
+* top 3 most popular patterns are:
+  * InnerSource Portal (6.5% of total traffic)
+  * 30 Day Warranty (4.5%)
+  * Standard Base Documentation (4%) // Core Team (4%)
+* TODO: Did pattern of the month have any noticeable effect here?
```

Comment:
```suggestion
* the mention of patterns in the Marketing Newsletters don't seem to have a significant effect on the performance (readership) of those patterns (more thorough analytics could be done here to confirm this).
```

Reply:
@robtuley I some very rough analytics here only.

I took the 3 patterns that were included in the board reports in this 3-month reporting period. Their performance is shown in this picture:

![pattern-performance-after-newsletter](https://user-images.githubusercontent.com/163029/203417153-a64ee9fa-a5eb-4c11-a46b-f61a9482280c.png)

Then I compared that to the performance of the same pictures in the 3-month period before that (i.e. before they were mentioned in the newsletter). Shown below.

![pattern-performance-before-newsletter](https://user-images.githubusercontent.com/163029/203416714-70f5e668-9cbe-4435-a3ed-260be1a98411.png)

To me it looks like the performance did not change significantly.

Reply:
@robtuley I was typing this bit while you were probably pushing/resolving some other content in the meantime.
So just mentioning you here again to make sure that you saw it, as though your angle wrt the impact of the Newsletter was really interesting.

Sadly no impact, I think.

Probably still good to think about how we could measure this more reliably :)
(I am not that proficient of a Google Analytics user, so if you know your way around there, i would be happy to give you access there, if I haven't already)